### PR TITLE
[Relay] Roundtrip part of pretty printer and parser

### PR DESCRIPTION
**Pull request by MarisaKirisame**
@joshpoll @srkreddy1238 @wweic can you guys review?

### CODE REVIEW

File: python/tvm/relay/grammar/py3/RelayLexer.tokens [hunk 1]
```diff
@@ -56,7 +56,7 @@ CNAME=40
 ':'=16
 'Tensor'=17
 '_'=18
-'v0.0.2'=19
+'v0.0.3'=19
```
Comment (jroesch) [inline python/tvm/relay/grammar/py3/RelayLexer.tokens]: Can you mark this file ignored in the .gitattributes

File: python/tvm/relay/grammar/py3/RelayParser.py [hunk 2]
```diff
@@ -176,7 +176,7 @@ class RelayParser ( Parser ):
 
     literalNames = [ "<INVALID>", "'('", "')'", "','", "'['", "']'", "'if'", 
                      "'else'", "'let'", "'='", "';'", "'{'", "'}'", "'fn'", 
-                     "'->'", "'def'", "':'", "'Tensor'", "'_'", "'v0.0.2'", 
+                     "'->'", "'def'", "':'", "'Tensor'", "'_'", "'v0.0.3'",
```
Comment (jroesch) [inline python/tvm/relay/grammar/py3/RelayParser.py]: mark this file too

File: src/relay/ir/alpha_equal.cc [hunk 3]
```diff
@@ -60,6 +60,19 @@ class AlphaEqualHandler:
       if (!rhs->derived_from<ExprNode>()) return false;
       return ExprEqual(Downcast<Expr>(lhs), Downcast<Expr>(rhs));
     }
+    if (const auto lhsm = lhs.as<ModuleNode>()) {
```
Comment (jroesch) [inline src/relay/ir/alpha_equal.cc]: Why are we alpha-equaling the whole module?

cc @zhiics

Comment (zhiics) [reply src/relay/ir/alpha_equal.cc]: I don't think I made this change. And I don't think we should do that as well.

File: cmake/modules/ANTLR.cmake [hunk 4]
```diff
@@ -14,21 +14,49 @@
...
+  find_program(ANTLR4 antlr4)
+
+  if (NOT ANTLR4)
+    file(GLOB_RECURSE ANTLR4JAR
+          /usr/local/lib/antlr-*-complete.jar
+          /usr/local/Cellar/*antlr-*-complete.jar)
+
+    # Get the first element of the list of antlr jars.
+    # Sort and reverse the list so the item selected is the highest
+    #   version in lib or else in Cellar if no lib installation exists.
```
Comment (zhiics) [inline cmake/modules/ANTLR.cmake]: align?

File: src/relay/ir/alpha_equal.cc (hunk 3 above)
Comment (MarisaKirisame) [reply src/relay/ir/alpha_equal.cc]: I need this to test roundtrip on whole module.
Also there is no reason why we dont want equality test on module (we can use this to test pure Module -> Module pass work as intended).

Comment (zhiics) [reply src/relay/ir/alpha_equal.cc]: Yeah, I have the same question here. Do we really need to compare the whole module?

Comment (zhiics) [reply src/relay/ir/alpha_equal.cc]: It sounds reasonable to me to have it for testing, although we don't have comparison of modules with multiple functions in most test cases.

File: cmake/modules/ANTLR.cmake (hunk 4 above)
Comment (MarisaKirisame) [reply cmake/modules/ANTLR.cmake]: WDYM? It is properly indented.

File: python/tvm/relay/grammar/py3/RelayLexer.tokens (hunk 1 above)
Comment (MarisaKirisame) [reply python/tvm/relay/grammar/py3/RelayLexer.tokens]: I had marked them as binary. Is it enough?

**Comment by MarisaKirisame** [comment]
@jroesch I add them as binary in .gitattributes. can you give some review and possibly merge it? it is blocking my other prs.

File: python/tvm/relay/grammar/py3/RelayLexer.tokens (hunk 1 above)
Comment (tqchen) [reply python/tvm/relay/grammar/py3/RelayLexer.tokens]: If this is an auxiliary file, we should delete it from the git and add to gitignore

Comment (MarisaKirisame) [reply python/tvm/relay/grammar/py3/RelayLexer.tokens]: it is deleted.

File: python/tvm/relay/grammar/py3/RelayParser.py (hunk 2 above)
Comment (MarisaKirisame) [reply python/tvm/relay/grammar/py3/RelayParser.py]: this file is needed.

**Pull request merged**